controllers/articles.controller.js

```diff
@@ -15,8 +16,11 @@ exports.getTopics = (req, res, next) => {
 
 exports.getArticlesById = (req, res, next) => {
```

Comment:
I think article (singular) would be a more appropriate term for the name of this controller